SPEC.md

```diff
@@ -399,20 +399,23 @@ JSON Schema for the Image Manifest (app image manifest, ACI manifest)
     ],
     "app": {
         "exec": [
-            "/usr/bin/reduce-worker"
+            "/usr/bin/reduce-worker",
+            "--quiet"
         ],
         "user": "100",
         "group": "300",
         "eventHandlers": [
             {
                 "exec": [
-                    "/usr/bin/data-downloader"
+                    "/usr/bin/data-downloader",
+                    "--enable-https"
                 ],
                 "name": "pre-start"
             },
             {
                 "exec": [
-                    "/usr/bin/deregister-worker"
+                    "/usr/bin/deregister-worker",
+                    "--verbose"
```

Comment:
Ok, I removed the --enable-https. So we can see that both app/exec and app/eventHandlers/exec can have parameters.